packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -840,6 +840,25 @@ const schema: SchemaType<"Posts"> = {
     hidden: !isLWorAF
   },
 
+  glossary: {
+    type: Array,
+    typescriptType: "GlossaryTerm[]",
+    optional: true,
+    nullable: true,
+    blackbox: true,
+    canRead: ['guests'],
+    control: "GlossaryEditForm",
+    group: formGroups.glossary,
+    canUpdate: [userOwns, 'admins'],
+    hidden: ({currentUser}) => !userCanCreateAndEditJargonTerms(currentUser)
+    // Implementation in postResolvers.ts
+  },
+  
+  'glossary.$': {
+    type: Object,
```

Comment:
As noted elsewhere, this should be a simple-schema definition that corresponds to the appropriate graphql schema.